### DISCUSSION TITLE
Repository for .deb Packages

### DISCUSSION THREAD

**Discussion by JokerGermany** [rustdesk/rustdesk#4518] [General]
I like Rustdesk and use it to support people on their phones.
But there is no good update possiblity on Ubuntu Mate 22.04, therfore i can't use it there.
Would be great if there would be a repository.

**Comment by csjoh** [comment]
+1 on this. An alternative would be to petition the package maintainers for Ubuntu/Debian to add the Rustdesk package to their official repositories.

**Reply by basilgello** [reply]
Official packaging can be done, but apart of time needed to package the dependencies, we need to get an approval from Debian Release Team to be able to push new Rustdesk versions in stable-updates. Or introduce an LTS release with minimum 3 years of support and backport only security and stability fixes.

Full disclosure: I am a Debian Maintainer for Kodi and I also made a new port of Xpra in Debian that was later rejected by original maintainer. Xpra took me a while to get properly packaged, and for RD it may take 2 months or even more. Would somebody fund this effort, I am ready to do this, but @rustdesk team is limited on resources and I am just an external contributor here using the software and maintaining F-Droid release I use the most.

**Reply by shreekant-sureliya** [reply]
i want to help for distributin of rustdesk through repository (mainly for ubuntu) let mw know how can i help ?

**Reply by xlionjuan** [reply]
I did

**Comment by xBr0th3rx** [comment]
We have set up a separate repository for packages that are not in the repository exactly for such things.

We put the updates in there and the clients also have our internal repository in the source.
